styles.css

```diff
@@ -0,0 +1,117 @@
+body {
+  background-image: url('header-shapes mobile@2x.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+header { /* HEADER CONTAINER */
+  display: flex;
+  height: 7vh;
+  justify-content: space-between;
+  margin-right: 6.4%;
+}
+
+.nav {
+  display: flex;
+  margin: 20%;
+}
+
+img {
+  max-width: 100%;
+}
+
+.menu {
+  /* border: 1px solid red; */

```

Comment:
- [ ] Please 🙏🏽 remove the commented line when it represents an unnecessary piece of code for the best practices of HTML & CSS.